lib/bin/cli.js

```diff
@@ -43,6 +43,8 @@ program.command('build')
   .action((command) => {
     // Set NODE_ENV to 'production'
     process.env.NODE_ENV = 'production'
+    process.env.PREFIX_LINKS = command.prefixLinks && 'prefix-links'
```

Comment:
It's setting process.env.PREFIX_LINKS to `'prefix-links'` if `command.prefixLinks` is `true` (actually not `undefined` but anyway). Since `process.env` doesn't accept attributes as boolean and converts `true` to `'true'` (mind the quotes), I thought it would be less confusing to do:

```javascript
if (process.env.PREFIX_LINKS === 'prefix-links') {
  // do stuff
}
```
rather than

```javascript
if (process.env.PREFIX_LINKS === 'true') {
  // do stuff
}
```

`process.env.PREFIX_LINKS` always passes the condition test, even when it's set to `'undefined'` (since it's a string).

By the way, maybe attaching `PREFIX_LINKS` to the `env` attribute is not the best thing to do, look at what `create-react-app` [is doing](https://github.com/facebookincubator/create-react-app/blob/fe7b5c212b5127775287ce444947f4c604c024dd/packages/react-scripts/config/env.js) for `env` variables.